### PR TITLE
Added support to windows

### PR DESCRIPTION
**Pull request by Th33x1l3**
- Add check of SO to load path
       If windows, use '!' as path list separator instead of ':'

- Updated spec tests to new minitest

### CODE REVIEW

File: lib/rconfig/load_paths.rb [hunk 1]
```diff
@@ -31,8 +31,14 @@ def add_load_path(path)
     # If the paths are made up of a delimited string, then parse out the
     # individual paths. Verify that each path is valid.
     def parse_load_paths(paths)
+      is_windows = Gem.win_platform?
+
       if paths.is_a? String
-        path_sep = (paths =~ /;/) ? ';' : ':'
+        if is_windows
```
Comment (rahmal) [inline lib/rconfig/load_paths.rb]: Move the get_path_sep into its own method. 

```Ruby
def get_pathe_sep
  is_windows = Gem.win_platform?
  ...
end

**Comment by rahmal** [comment]
@Th33x1l3 first off, I apologize for not getting to this in time to help you. Second, great job! I had something similar in mind, but never got to it. This looks good to me, but I think you should make that one change I mentioned. Also, Update the minor version and the ChangeLog.

**Comment by Th33x1l3** [comment]
Next monday i take care if it

via CloudMagic Email [https://cloudmagic.com/k/d/mailapp?ct=pa&cv=8.6.37&pv=6.0.1&source=email_footer_2] On Fri, Aug 19, 2016 at 7:16 PM, Rahmal Conda < notifications@github.com [notifications@github.com] > wrote:
@Th33x1l3 [https://github.com/Th33x1l3] first off, I apologize for not getting to this in time to help you. Second, great job! I had something similar in mind, but never got to it. This looks good to me, but I think you should make that one change I mentioned. Also, Update the minor version and the ChangeLog.

**Comment by rahmal** [comment]
@Th33x1l3 Great, I'll pull it in and release a new version then.

**Comment by Th33x1l3** [comment]
Ok commit is done
As you can see from changeset i had to change gem spec dependencies so json gem is installed in version greater than 1.8.1 so we people with ruby version equal or greater than 2.2.x can make changes to it.

**Comment by rahmal** [comment]
Well done! Reviewed the changes. Looks good. Merging and publishing now. I will let you know when its done.

**Comment by rahmal** [comment]
@Th33x1l3 RConfig 0.5.4 has been released to [RubyGems.org](https://rubygems.org/gems/rconfig).

**Pull request merged**